src/effects/specialeffectchainslots.cpp

```diff
@@ -114,6 +114,7 @@ QuickEffectChainSlot::QuickEffectChainSlot(const QString& group,
         addEffectSlot(formatEffectSlotGroup(group, i));
         m_effectSlots.at(i)->setEnabled(true);
     }
+    m_pControlNumPresetsAvailable->set(m_pChainPresetManager->numQuickEffectPresets());
```

Comment:
redundant if the base class uses `numPresets()` for this

Reply:
No. When the base class constructor runs, the subclass doesn't exist yet which means the `numPresets` implementation of the base class is called.